### PR TITLE
General improvements

### PR DESCRIPTION
**Pull request by pevalme**
* Updated main README according to the improvements that has been done over mist. This README also inform about the compatibility of the tool with different OS and the external tools needed.
* Added a README in the folder *scripts* to inform about their usage
* Csv files generated when executing the script *run_benchmarks* with *--graph* option have the right extension
* The *.html* file obtained when generating graphs is called *index.html*
* Improved the help menu of the algorithms by giving more useful names to the arguments
* The script *run_benchmarks* let us choose which algorithm will be used with any option
* Changed from *wc -l* + *head -n*  to several uses of *sed*
* Changed from *i=`expr $i +1`* to *i=$[i + 1]*
* Improved mist output to show infinite symbol instead of a negative integer

### CODE REVIEW

**Comment by pierreganty** [comment]
@pevalme any good reason to remove some regression tests in this PR?

**Comment by pevalme** [comment]
I don't know why are these changes here but the removed folder is the older one so it shouldn't cause any change in your repository.

I'll check if there is any way to remove the useless changes.

File: README.md [hunk 1]
```diff
@@ -42,7 +48,14 @@ Please report issues on [GitHub][issueslist].
 
 Contact the author, [Pierre Ganty][email].
 
+## Requeriments
```
Comment (pierreganty) [inline README.md]: spell check: Requirements

File: README.md [hunk 2]
```diff
@@ -42,7 +48,14 @@ Please report issues on [GitHub][issueslist].
...
+
+To use the scripts it's needed [mustache][mustachepage] to be installed.
```
Comment (pierreganty) [inline README.md]: Fix english: "You need to install mustache to generate graphs" (or something like this)

File: README.md [hunk 3]
```diff
@@ -42,7 +48,14 @@ Please report issues on [GitHub][issueslist].
...
+
+The tool works for Linux and MacOS
```
Comment (pierreganty) [inline README.md]: for -> under

**Comment by pierreganty** [comment]
@pevalme can you make changes so that the `merge pull request` button gets enabled?

**Comment by pierreganty** [comment]
:+1: LGTM

**Pull request merged**